x-pack/docs/en/watcher/release-notes.asciidoc

```diff
@@ -201,7 +201,7 @@ October 28, 2015
   they did before the upgrade. For example if `watcher.dynamic_indices.time_zone`
   setting was set to `+01:00` and a watch has the following index name
   `<logstash-{now/d}>` then after the upgrade you need to update this watch to
-  use the following index name `<logstash-{now/d{YYYY.MM.dd|+01:00}}>`.
+  use the following index name `<logstash-{now/d{yyyy.MM.dd|+01:00}}>`.
```

Comment:
these are old release notes, that might be outdated in other ways as well, I am not sure if it is worth updating?